api/spec/src/notification.tsp

```diff
@@ -8,6 +8,468 @@ using TypeSpec.OpenAPI;
+
+  /**
+   * Webhook URL where the notification is sent.
+   */
+  @summary("URL")
+  @example("https://example.com/webhook")
+  @visibility("read", "create", "query", "update")
+  url: string;
+
+  // FIXME: example
+  /**
+   * Custom HTTP headers sent as part of the webhook request.
+   */
+  @summary("Custom HTTP Headers")
+  @visibility("read", "create", "query", "update")
+  customHeaders?: Record<string>;
+
+  /**
+   * Signing secret used for webhook request validation on the receiving end.
+   *
+   * Format: `base64` encoded random bytes optionally prefixed with `whsec_`. Recommended size: 24
+   */
+  @summary("Signing Secret")
+  @pattern("^(whsec_)?[a-zA-Z0-9+/=]{32,100}$")
+  @example("whsec_S6g2HLnTwd9AhHwUIMFggVS9OfoPafN8")
+  @visibility("read", "create", "query", "update")
+  signingSecret: string;
+}
+
+/**
+ * Notification channel.
+ */
+@friendlyName("NotificationChannel")
+@discriminator("type")
+@oneOf
+union NotificationChannel {
+  webhook: NotificationChannelWebhook,
+}
+
+/**
+ * Union type for requests creating new notificatio nchannel with certain type.
+ */
+@friendlyName("NotificationChannelCreateRequest")
+@discriminator("type")
+@oneOf
+union NotificationChannelCreateRequest {
+  webhook: NotificationChannelWebhook,
+}
+
+/**
+ * Request with input parameters for creating new notification channel with webhook type.
+ */
+@friendlyName("NotificationChannelWebhookCreateRequest")
+@withVisibility("create", "update")
+model NotificationChannelWebhookCreateRequest{
+  ...NotificationChannelWebhook;
+}
+
+/**
+ * Order by options for notification channels.
+ */
+@friendlyName("NotificationChannelOrderBy")
+enum NotificationChannelOrderBy {
+  #suppress "@openmeter/api-spec/casing" "Ignore due to backward compatibility"
+  id: "id",
+  #suppress "@openmeter/api-spec/casing" "Ignore due to backward compatibility"
+  type: "type",
+  #suppress "@openmeter/api-spec/casing" "Ignore due to backward compatibility"
+  createdAt: "createdAt",
+  #suppress "@openmeter/api-spec/casing" "Ignore due to backward compatibility"
+  updatedAt: "updatedAt",
+}
+
+@route("/api/v1/notification/channels")
 @tag("Notification (Experimental)")
-interface Notification {}
+interface NotificationChannels {
+  /**
+   * List all notification channels.
+   */
+  @get
+  @operationId("listNotificationChannels")
+  @summary("List notification channels")
+  list(
+    /**
+     * Include deleted notification channels in response.
+     *
+     * Usage: `?includeDeleted=true`
+     */
+    @query
+    @example(true)
+    includeDeleted?: boolean,
+
+    /**
+     * Include disabled notification channels in response.
+     *
+     * Usage: `?includeDisabled=false`
+     */
+    @query
+    @example(false)
+    includeDisabled?: boolean,
+
+    ...OpenMeter.QueryPagination,
+    ...OpenMeter.QueryOrdering<NotificationChannelOrderBy>,
+  ): OpenMeter.PaginatedResponse<NotificationChannel> | OpenMeter.CommonErrors;
```

Comment:
Since it's in the same namespace, you don't need `OpenMeter.` I believe.

Reply:
Notification has itw on namespace now.